src/server/publicApi.js

```diff
@@ -56,19 +56,31 @@ const runPublicApi = (gameServer) => {
     };
     });
 
-    router.get('/model/:id', async ctx => {
+    router.get('/model/:game/:id/:secret', async ctx => {
```

Comment:
In general, it is not a good practice to put secrets as query parameters into REST endpoints. The reason is that the full requested URLs are usually written down in the webservers logs and can leak from there...

Could these three numbers be sent in the HTTP body (i.e. in JSON)? Or alternatively, put them into a custom HTTP header as described here: https://www.fullcontact.com/blog/2016/04/29/never-put-secrets-urls-query-parameters/